src/services/authService.js

```diff
@@ -29,4 +31,47 @@ export default class AuthService {
 
     return { token, ...Helper.omitFields(user, ['password']) };
   }
+
+  /**
+ *
+ * @param {string} email email address
+ * @returns {string} token
+ */
+  static async forgotPassword(email) {
+    const result = await User.findOne({ where: { email } });
+    if (!result) throw new Error('Email not found');
+    const { dataValues: { first_name: name } } = result;
+    const token = await jwt.sign({
+      email
+    }, jwtSecret, { expiresIn: '1h' });
+    const emailDetails = {
+      receiver: email,
+      sender: 'chubi.best@gmail.com',
```

Comment:
The sender details should be stored in the `.env` file because it can be changed.